.travis.yml

```diff
@@ -53,6 +53,7 @@ before_script:
   - echo no | android create avd --force -n test -t android-19 --abi armeabi-v7a
   - emulator -avd test -no-audio -no-window &
   - sleep 300
+  - android-wait-for-emulator
```

Comment:
sleep is used instead, android-wait-for-emulator is still buggy and can result in test failures, adding both will work but will waste time.